Practice2/src/styles/variables/main-var.css

```diff
@@ -0,0 +1,4 @@
+:root {
+  --black-dark: #4B4B4B;
```

Comment:
You have a lot of variables in design. Please list all variables here in following groups:
- color
- font-size
- padding
- margin
- font-weight
- gap
- ...
-

Reply:
@duylong13 Pls double-check your code. I still found some colours that haven't been defined with a variable yet. E.g. `input-stay-in-touch.css`, line 3 with the black colour.

Reply:
Fixed